packages/core/src/components/Typography/FontSizesSection.tsx

```diff
@@ -16,31 +16,11 @@ export function FontSizesSection({
     onOpen: onAddVariantModalOpen,
     onClose: onAddVariantModalClose,
   } = useDisclosure()
-  const sortFontSizes = function () {
-    const rem: TFontSizeVariant[] = []
-    const px: TFontSizeVariant[] = []
-    const em: TFontSizeVariant[] = []
 
-    fontSizeData.map((font) => {
-      if (font.unit == 'rem') {
-        rem.push(font)
-      }
-      if (font.unit == 'em') {
-        em.push(font)
-      }
-      if (font.unit == 'px') {
-        px.push(font)
-      }
-    })
-
-    rem.sort((a, b) => a.value - b.value)
-    em.sort((a, b) => a.value - b.value)
-    px.sort((a, b) => a.value - b.value)
-
-    return rem.concat(px, em)
-  }
-
-  fontSizeData = sortFontSizes()
+  fontSizeData.sort((a, b) => {
+    if(a.unit === b.unit) return a.value - b.value
+    else return a.unit < b.unit ? 1 : (a.unit > b.unit ?  -1 : 0)
```

Comment:
Hey @Bismay5467, but we need it to sort rem, then em, then px. Otherwise, 1.5 px and 1.6 rem will appear sorted.